### ISSUE TITLE
RuntimeError: vagrant-share can't be installed without vagrant-login

### ISSUE THREAD

**Issue by ngdinhtoan** [hashicorp/vagrant#5050]
Acctually, this error appear when I type some incorrect command.

``` bash
$ vagrant help
/opt/vagrant/embedded/gems/gems/vagrant-share-1.1.2/lib/vagrant-share/activate.rb:8:in `rescue in <encoded>': vagrant-share can't be installed without vagrant-login (RuntimeError)
    from /opt/vagrant/embedded/gems/gems/vagrant-share-1.1.2/lib/vagrant-share/activate.rb:5:in `<encoded>'
    from /opt/vagrant/embedded/gems/gems/vagrant-share-1.1.2/lib/vagrant-share/activate.rb:16:in `RGLoader_load'
    from /opt/vagrant/embedded/gems/gems/vagrant-share-1.1.2/lib/vagrant-share/activate.rb:16:in `<top (required)>'
    from /opt/vagrant/embedded/gems/gems/vagrant-share-1.1.2/lib/vagrant-share.rb:23:in `require'
    from /opt/vagrant/embedded/gems/gems/vagrant-share-1.1.2/lib/vagrant-share.rb:23:in `block in <class:Plugin>'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.0/lib/vagrant/cli.rb:75:in `call'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.0/lib/vagrant/cli.rb:75:in `block (2 levels) in help'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.0/lib/vagrant/registry.rb:48:in `block in each'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.0/lib/vagrant/registry.rb:47:in `each'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.0/lib/vagrant/registry.rb:47:in `each'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.0/lib/vagrant/cli.rb:69:in `block in help'
    from /opt/vagrant/embedded/lib/ruby/2.0.0/optparse.rb:885:in `initialize'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.0/lib/vagrant/cli.rb:57:in `new'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.0/lib/vagrant/cli.rb:57:in `help'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.0/lib/vagrant/cli.rb:32:in `execute'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.0/lib/vagrant/environment.rb:301:in `cli'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.0/plugins/commands/help/command.rb:11:in `execute'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.0/lib/vagrant/cli.rb:42:in `execute'
    from /opt/vagrant/embedded/gems/gems/vagrant-1.7.0/lib/vagrant/environment.rb:301:in `cli'
    from /opt/vagrant/bin/../embedded/gems/gems/vagrant-1.7.0/bin/vagrant:174:in `<main>'
```

I think it should not output like this.

**Comment by lonniev** [comment]
Nicely understated @ngdinhtoan. You are right: foobarring a utility's little help message is not something desirable. ;-)

It's an effective way to force users to update, though!

**Comment by ngdinhtoan** [comment]
Upgraded :)

**Comment by awalls** [comment]
Was getting this error on 1.6.5. Upgraded to latest version 1.7.2 which appears to have fixed the problem.

**Comment by exuperok** [comment]
fixed forme as well after upgrading to 1.7.2
